src/pages/MeetingPage/MeetingPage.tsx

```diff
@@ -15,12 +15,70 @@ import NotFoundPage from '../NotFoundPage'
 const StyledForm = styled.form`
   padding: 48px 24px;
 `
+
+const GetPropertiesAndCategories = gql`
+  query GetPropertiesAndCategories {
+    property(where: { type: { _eq: "member" } }) {
+      id
+      name
+    }
+    category(where: { class: { _eq: "member" } }) {
+      id
+      name
+    }
+  }
+`
+
+const UpdateMemberCreated = gql`
+  mutation UpdateMemberCreated($memberId: String!) {
+    update_member_by_pk(pk_columns: { id: $memberId }, _set: { created_at: "NOW()" }) {
+      id
+      username
+    }
+  }
+`
+
+const UpdateMemberProperties = gql`
+  mutation UpdateMemberProperties($memberPropertiesInput: [member_property_insert_input!]!) {
+    insert_member_property(
+      objects: $memberPropertiesInput
+      on_conflict: { constraint: member_property_member_id_property_id_key, update_columns: [value] }
+    ) {
+      affected_rows
+    }
+  }
+`
+
+const InsertMemberTask = gql`
+  mutation InsertMemberTask($currentMemberId: String!, $managerId: String!, $taskTitle: String!) {
+    insert_member_task(
+      objects: {
+        member_id: $currentMemberId
+        author_id: $managerId
+        executor_id: $managerId
+        title: $taskTitle
+        due_at: "now()"
+      }
+    ) {
+      affected_rows
+    }
+  }
+`
+
+const GetMemberByUsername = gql`
+  query GetMemberByUsername($appId: String!, $username: String!) {
+    member_public(where: { app_id: { _eq: $appId }, username: { _eq: $username } }) {
+      id
+    }
+  }
+`
+
 const MeetingPage = () => {
-  const { settings } = useApp()
-  const { currentMemberId } = useAuth()
+  const { id: appId, settings } = useApp()
+  const { currentMemberId, currentMember } = useAuth()
   const { username: managerUsername } = useParams<{ username: string }>()
```

Comment:
![image](https://github.com/urfit-tech/lodestar-app/assets/64723534/c58f140e-bc2a-4221-891d-66d525ed47cb)

Do we need to check for the `@` prefix or not ?

Reply:
I think instead of just paying attention to the '@' suffix, it might to verify if it's business by checking?
Because here we're looking up the userName.

Reply:
But we use the URL parameters to check if it's business, and whether it has the '@' prefix will affect our verification.

Code:
```
  const { username: managerUsername } = useParams<{ username: string }>()
  
  const { data: memberData } = useQuery<hasura.GetMemberByUsername, hasura.GetMemberByUsernameVariables>(
    GetMemberByUsername,
    {
      variables: { appId, username: managerUsername },
    },
  )

  const managerId = managerUsername ? memberData?.member_public[0]?.id || undefined : undefined
  ```